### ISSUE TITLE
DJ Fenris: ID selection menu does not disappear

### ISSUE THREAD

**Issue by progdormedp** [mtgred/netrunner#3831]
When installing DJ Fenris with Hayley (as the first install). 
Probably related to her install prompt?

**Comment by Saintis** [comment]
This might be an issue with the resent move to Reagent.

**Comment by chaosof99** [comment]
I've run into the same issue today. Had to play half of a game with a fenris prompt stuck on screen.

![fenrishaley](https://user-images.githubusercontent.com/44696340/50386922-abc1ca80-06f1-11e9-979a-a7b2dcd671d4.PNG)

**Comment by NoahTheDuke** [comment]
Did /close-prompt not work?

**Comment by chaosof99** [comment]
No. Attempted multiple times. Also tried /undo-click and /undo-turn, which only made things worse as you can see. Lead to multiple overlapping Fenris prompts being displayed,

**Comment by x3N1GM4x** [comment]
FYI, someone in my local meta had this problem last night (DJ Fenris into MaxX as Hayley) and they said a page refresh with F5 cleared it for them. No idea if that helps at all!
